tailored-css.md

```diff
@@ -0,0 +1,120 @@
+# Tailored CSS
+
+## The site should be fast
+
+We value site speed a lot at SpanishDict.com. As we've started expanding our
+site into Latin America, this has become even more important. Connection speeds
+tend to be slower than in the USA, so optimizing for them is always on our mind.
+
+In researching how to improve our site's speed, a recurring theme is that
+[perceived speed tends to be more important than actual speed](http://blog.teamtreehouse.com/perceived-performance).
+We're always on the lookout for ways to make the site _feel_ faster.
+
+## State of CSS at SpanishDict
+
+One of the things that can slow down how a site feels (it certainly did for us)
+is blocking CSS. When you add a `link` tag, the browser has to first load and
+parse the entire file, because it doesn't know how to render the content without
+it.
+
+Our CSS is not well organized, and as a result our css files are large. For
```

Comment:
A little context might reflect better on how things got to this state: we followed the common pattern of bundling all CSS into one asset, since one request, even a fairly large one, is generally faster than multiple small ones when using HTTP 1.1 (which was all that was around at the time). At some point over the past four years of adding features and new parts to the product, our CSS grew so large that it became a liability.